docs/user-guide/services-firewalls.md

```diff
@@ -6,6 +6,32 @@ exposure to the internet.  When exposing a service to the external world, you ma
 one or more ports in these firewalls to serve traffic.  This document describes this process, as
 well as any provider specific details that may be necessary.
 
+### Restrict Access For LoadBlancer Service
+
+ When using a Service with `spec.type: LoadBalancer`, you can specify the IP ranges that are allowed to access the load balancer
+ by using `spec.loadBalancerSourceRanges`. This field takes a list of IP CIDR ranges, which Kubernetes will use to configure firewall exceptions.
+ This feature is currently supported on Google Compute Engine, Google Container Engine and AWS. This field will be ignored if the cloud provider does not support the feature.
+
+ In the following example, a load blancer will be created that is only accessible to clients with IP addresses ranging from 10.180.0.0/16 to 10.245.0.0/24.
```

Comment:
this sentence is confusing, makes it sound like we're saying 10.180.0.0/16 -> 10.245.0.0/24 like start and end, when we actually mean the 65,000 ips in the /16 and the 256 in the /24, right?